### PR TITLE
Minor changes for matrix plugins

### PR DESCRIPTION
**Pull request by scarabus**

### CODE REVIEW

File: Libs/RaptureCore/src/main/java/rapture/mail/SMTPConfig.java [hunk 1]
```diff
@@ -34,39 +34,50 @@ public String getUsername() {
...
         this.username = username;
+        return this;
     }
 
     public String getPassword() {
         return password;
     }
 
-    public void setPassword(String password) {
+    public SMTPConfig setPassword(String password) {
```
Comment (jonathan-major) [inline Libs/RaptureCore/src/main/java/rapture/mail/SMTPConfig.java]: why return the config from each method?

Comment (scarabus) [inline Libs/RaptureCore/src/main/java/rapture/mail/SMTPConfig.java]: Let's you do daisy chaining (AKA decorator pattern):
add(new SMTPConfig().setPassword(foo).setUser(bar).setPort(1));
I'd like to change the Antlr code to do that to all the auto-generated stuff

File: Libs/RaptureCore/src/main/java/rapture/kernel/AdminApiImpl.java [hunk 2]
```diff
@@ -453,14 +448,10 @@ public void emailUser(CallingContext context, String userName, String emailTempl
         RaptureUser user = getUser(context, userName);
         if (user != null) {
             templateValues.put("user", user);
-            Mailer.email(context, emailTemplate, templateValues);
-        } else {
-            throw RaptureExceptionFactory.create(HttpURLConnection.HTTP_BAD_REQUEST, adminMessageCatalog.getMessage("NoExistUser", userName)); //$NON-NLS-1$
         }
+        Mailer.email(context, emailTemplate, templateValues);
```
Comment (jonathan-major) [inline Libs/RaptureCore/src/main/java/rapture/kernel/AdminApiImpl.java]: 1. what happens if user is null?
2. The class uses RaptureExceptionFactory alot so does it make sense to change it for this one case?

Comment (scarabus) [inline Libs/RaptureCore/src/main/java/rapture/kernel/AdminApiImpl.java]: In my test case I didn't have a user so it barfed unnecessarily. 
This was a quick fix; I can fix my test to add a user.

Comment (jonathan-major) [inline Libs/RaptureCore/src/main/java/rapture/kernel/AdminApiImpl.java]: ok do that and commit again. thanks

**Pull request merged**